admin/config.js

```diff
@@ -23,7 +23,8 @@ module.exports = {
   TIME_BETWEEN_QUESTIONS: 2,
   LINES_PER_CHECK_FORM: 25,
   Data: {
-    allowedWords: process.env.ALLOWED_WORDS || 'aaa,bcd,dcd,tfg,bxx'
+    allowedWords: process.env.ALLOWED_WORDS || 'aaa,bcd,dcd,tfg,bxx',
+    pinSubmissionMaxAttempts: process.env.PIN_SUBMISSION_MAX_ATTEMPTS || 15
```

Comment:
I am struggling with the concept of max tries given that there are 10K possible pin combinations and fewer pupils than that in each school